pkg/apis/sealed-secrets/v1alpha1/types.go

```diff
@@ -20,7 +20,8 @@ const (
 
 // SealedSecretSpec is the specification of a SealedSecret
 type SealedSecretSpec struct {
-	Data []byte `json:"data"`
+	Data          []byte            `json:"data"`
```

Comment:
I'm not really good at writing comments, any suggestions what the deprecated comment should say?

Reply:
"Data is deprecated and will be removed eventually. Use per-value EncryptedData instead."

It doesn't have to say much, just give the reader an indication that the reason both exist is because we're transitioning, and they they should use EncryptedData for new code.